### PR TITLE
Add Xing support to existing providers

### PR DESCRIPTION
**Pull request by damianz5**
ping @SylvainGuittard @kamilmusial @clash82

### CODE REVIEW

**Comment by damianz5** [comment]
ping @SylvainGuittard @kamilmusial @clash82

File: Resources/views/default/xing.html.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% block share_buttons_service %}
+    <div data-type="xing/share"{% if shape != 'none' %} data-shape="{{ shape }}"{% endif %}{% if counter != 'none' %} data-counter="{{ counter }}"{% endif %}{% if language != 'none' %} data-lang="{{ language }}"{% endif %}></div>
+    <script>
+        {% autoescape 'js' %}
+            ;(function (d, s) {
+                var x = d.createElement(s),
+                        s = d.getElementsByTagName(s)[0];
```
Comment (clash82) [inline Resources/views/default/xing.html.twig]: too much indents here :)

**Comment by SylvainGuittard** [comment]
+1

**Comment by clash82** [comment]
+1

**Comment by kamilmusial** [comment]
+1

**Pull request merged**